quixstreams/models/topics/manager.py

```diff
@@ -211,6 +211,9 @@ def _get_source_topic_config(
             topic_name
         ] or deepcopy(self._non_changelog_topics[topic_name].config)
 
+        if topic_config is None:
+            raise RuntimeError(f"No configuration can be found for topic {topic_name}")
+
```

Comment:
When exactly can this happen?

Reply:
`admin.inspect_topics` returns `{"topic-name": None}` if the topic doesn't already exist. 

While I don't think we can get to the exception using the app if you directly use the topic manager it's possible by calling `repartition_topic()` or `changelog_topic()` before registering the "main" topic.